Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean

```diff
@@ -86,7 +86,8 @@ theorem spec_of_surjective {R S : CommRingCat} (f : R ⟶ S) (h : Function.Surje
     IsClosedImmersion (Spec.map f) where
   base_closed := PrimeSpectrum.closedEmbedding_comap_of_surjective _ _ h
   surj_on_stalks x := by
-    erw [← localRingHom_comp_stalkIso, CommRingCat.coe_comp, CommRingCat.coe_comp]
+    rw [← Scheme.localRingHom_comp_stalkIso]
+    erw [CommRingCat.coe_comp, CommRingCat.coe_comp]
```

Comment:
Or maybe there are better formalizations of the LHS of `Scheme.localRingHom_comp_stalkIso` that gets rid of these?

Reply:
I got rid of the `erw`s, but with a different technique that I think is more useful in general.